### ISSUE TITLE
[Process] more frequent, smaller merges

### ISSUE THREAD

**Issue by lixuna** [cncf/cnf-conformance#207]
### [Process] big merges are problematic, start using more frequent, smaller merges

Tasks: 

- [x] update process
- [ ] update documentation

**Comment by lixuna** [comment]
Final decision:
---

- [ ] Merge based on feature branches into master
- [ ] Gate cannot merge unless it passes tests
- [ ] Do not merge to master until tests are passing
- [ ] Gate implementation:
- [ ] Starting as guideline
- [ ] Person who created PR does not merge
- [ ] Person reviewing PR only merges if all tests are passing
- [ ] Person merging is responsible for fixing the `master` if the merge breaks things. (eg. revert build)
- [ ] Stop using develop branch
- [ ] Keep using A/C - 2 people are responsible for the code going to master
- [ ] Feature branch naming can be deferred to individual issue planning meetings

**Comment by lixuna** [comment]
Closing this, as we have a related documentation ticket #26
